src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -300,23 +308,30 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
    * True if date is >= the minimum (ie, today) and <= the maximum (ie, the 10th day).
    * This is used for Vue form validation (in Date Rules above).
    */
-  isBetweenDates (minDate: string, maxDate: string, dateStrToValidate: string): boolean {
+  private isValidDateRange (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
     if (!dateStrToValidate) { return true }
-    return (new Date(dateStrToValidate) >= new Date(minDate)) && (new Date(dateStrToValidate) <= new Date(maxDate))
+    // only compare year/month/day (ignore time)
+    let date = new Date(dateStrToValidate)
+    date = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate())
+    const minDay = new Date(minDate.getFullYear(), minDate.getMonth(), minDate.getDate())
+    const maxDay = new Date(maxDate.getFullYear(), maxDate.getMonth(), maxDate.getDate())
+    return (date >= minDay && date <= maxDay)
```

Comment:
I'm a bit nervous when I see `Date()` used outside of the date mixin (in which much thought/testing was given to things). It's probably OK here because you're comparing results.

I'm a bit nervous that you're mixing UTC and non-UTC dates here. You may get unexpected results for the day if it's after 4pm or 5pm Pacific (depending on daylight time).

I'd prefer to see date methods in Date Mixin.

You may also find that you can use existing Date Mixin methods to compare dates (eg, convert to yyyy-mm-dd and then just apply string >= and <=).

Right now I don't have that warm, fuzzy feeling that this code works correctly in all cases.

Reply:
This is directly copied from `EffectiveDateTime` 

I'd rather use a date library personally, moment is depreciated, maybe something like `date-fns`.

Reply:
Modern JS should not need a date/time library. The main problem is devs not understanding how UTC time works --hence our Date Mixin. We can talk more about this if you like; I'm willing to change my mind :)

How about leaving EffectiveDateTime.vue alone and moving your code to the Date Mixin? This should result in an overall code simplification/improvement.

Reply:
I'm OK with the new code.